methods/arrays/chunk.js

```diff
@@ -0,0 +1,7 @@
+function chunk(array, size) {
+  if (array.length <= size) {
+    return [array];
+  }
+  return [array.slice(0, size), ...chunk(array.slice(size), size)];
```

Comment:
I suppose .slice method is forbidden 
![image](https://github.com/Alex-prokop/unit_tests/assets/22891941/e3ed6654-fdc2-42cf-a74a-730370753b15)